config/software/runit.rb

```diff
@@ -15,18 +15,15 @@
 #
 
 name "runit"
-default_version "2.1.1"
+default_version "2.1.2"
 
 license "BSD-3-Clause"
 license_file "../package/COPYING"
 skip_transitive_dependency_licensing true
 
-version "2.1.2" do
-  source md5: "6c985fbfe3a34608eb3c53dc719172c4"
-end
-version "2.1.1" do
-  source md5: "8fa53ea8f71d88da9503f62793336bc3"
-end
+# versions_list: http://smarden.org/runit/ filter=*.tar.gz
```

Comment:
I think we mentioned using `do_not_auto_update` since there isn't a formal list of all the releases

Reply:
yeah, that's right